modules/3d-tiles/docs/api-reference/tiles-3d-loader.md

```diff
@@ -6,16 +6,16 @@
 
 Parses a [3D tiles](https://github.com/AnalyticalGraphicsInc/3d-tiles) tileset.
 
-| Loader                | Characteristic                                                                                                     |
-| --------------------- | ------------------------------------------------------------------------------------------------------------------ |
-| File Extensions       | `.b3dm`,`.i3dm`, `.pnts`, `.cmpt`                                                                                  |
-| File Type             | Binary (with linked assets)                                                                                        |
-| File Format           | [3D Tiles](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#tile-format-specifications) |
-| Data Format           | [Data Formats](#data-formats)                                                                                      |
-| Decoder Type          | Asynchronous                                                                                                       |
-| Worker Thread Support | No                                                                                                                 |
-| Streaming Support     | No \*                                                                                                              |
-| Subloaders            | `DracoLoader` (`.pnts`), `GLTFLoader` (`.b3dm`, `.i3dm`), `ImageLoader` (.`.jpg`, `.png`), `TextureLoader` (`.ktx`)                                                           |
+| Loader                | Characteristic                                                                                                      |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| File Extensions       | `.b3dm`,`.i3dm`, `.pnts`, `.cmpt`                                                                                   |
```

Comment:
Are we adding glb support here?
```suggestion
| File Extensions       | `.b3dm`,`.i3dm`, `.pnts`, `.cmpt`, `.glb`, `.gltf`                                                                                   |
```

Reply:
Let's follow the spec. if the spec only support glb let's do that, if the spec supports both lets aim for that.